### ISSUE TITLE
build_choose_clause unicode issue

### ISSUE THREAD

**Issue by ryphill** [sidecars/python-quickbooks#46]
I've recently run into an issue doing `Customer.choose` with unicode strings which stem from the `build_choose_clause` function. If I pass in a unicode string with a unicode specific character it will throw a `UnicodeEncodeError` when it tries to format the string. This would probably happen with `build_where_clause` as well since it works similarly.

I'm on python 2.7.12 using package version 0.5.1.

Example the triggers the problem:

``` python
choices = [u'Some Company – Location']
build_choose_clause(choices, u'DisplayName')

python2.7/site-packages/quickbooks/utils.pyc in build_choose_clause(choices, field)
     27         for choice in choices:
     28             if isinstance(choice, six.string_types):
---> 29                 where.append("'{0}'".format(choice.replace("'", "\'")))
     30             else:
     31                 where.append("{0}".format(choice))

UnicodeEncodeError: 'ascii' codec can't encode character u'\u2013' in position 24: ordinal not in range(128)
```

**Comment by harel** [comment]
I'm having a similar problem on Python 2.7, using latest package 0.6.1, calling the `where` method with a unicode filter string. The `where` call in the mixin drops the same exception of UnicodeEncodeError.

Is there a solution to this?

**Comment by deathnfudge** [comment]
Fixed by Pull Request #88
